### DISCUSSION TITLE
Search for exact matches with quotation marks

### DISCUSSION THREAD

**Discussion by shrugal** [searxng/searxng#3467] [Q&A]
Is there a way to search for exact matches, like using "quotation marks" or something? It works in almost every search engine I know of, but apparently not when using SearXNG.

**Comment by unixfox** [comment]
It depends on the engine used. This will work for google engine but not for the others.

Searxng is a meta search engine not a search engine by itself so it can't filter to just the keywords that you want.

**Reply by David-Ray** [reply]
Then is there a way to exclude results that are not exact matches? or exclude search engines will not give exact matches? 

For instance, I'm searching for a term "revolutionayem" and all the results I see are for "revolutionary". I suppose if I page down long enough, I might see a different result. But for how many days would I have to wade through bad results?
